src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -301,6 +302,17 @@ function MoneyRequestConfirmationList({
     // TODO: remove the !isTypeInvoice from this condition after BE supports tax for invoices: https://github.com/Expensify/App/issues/41281
     const shouldShowTax = isTaxTrackingEnabled(isPolicyExpenseChat, policy) && !isTypeInvoice;
 
+    useEffect(() => {
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+        if (isEmptyObject(policy) || !shouldShowTax || transaction?.taxCode || !taxRates) {
+            return;
+        }
+        const defaultTaxCode = TransactionUtils.getDefaultTaxCode(policy, transaction);
+        if (defaultTaxCode) {
+            IOU.setMoneyRequestTaxRate(transactionID, defaultTaxCode);
+        }
```

Comment:
I think we could remove this condition - if (defaultTaxCode)?
we can fallback to empty string and always set tax rate